src/util.py

```diff
@@ -385,3 +386,11 @@ def make_required_dirs(path: str):
     """
     out_path = Path(path).parent
     out_path.mkdir(parents=True, exist_ok=True)
+
+def add_rank_column(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Add a column of 1s to the dataframe
+    @param df: the dataframe to add the rank column of 1s to
+    """
+    df['rank'] = 1
```

Comment:
if we are going to show the headers in the future (because we have a bunch of files that look the same but have different meanings), most of the other algorithms have 'Rank' for the column name for the rank column added. It's a minor thing, but I think this can be changed to also use 'Rank' for uniformity. Also since this is being added, we should update the rest of the algorithms that are manually adding a rank column of 1s to use this function.